client/components/Recipe/RecipesPage.vue

```diff
@@ -1,68 +1,92 @@
 <!-- Default page that also displays freets -->
 
 <template>
-    <main>
-      <AddRecipeForm />
-      <hr>
-      <h2 class="display-4">Recipes</h2>
-      <FilterRecipeForm />
-      <br>
-      <div class="font-italic">
-        <h2 v-if="$store.state.keyword && $store.state.ingredients.length">
-          Displaying recipes with keyword {{ $store.state.keyword }} and ingredient(s) {{ $store.state.ingredients.join(', ') }}
-        </h2>
-        <h2 v-else-if="$store.state.keyword">
-          Displaying recipes with keyword {{ $store.state.keyword }}
-        </h2>
-        <h2 v-else-if="$store.state.ingredients.length">
-          Displaying recipes with ingredient(s) {{ $store.state.ingredients.join(', ') }}
-        </h2>
-        <h2 v-else>Displaying all recipes</h2>
-      </div>
-      <section
-        v-if="$store.state.recipes.length"
-      >
-        <RecipeComponent
-          v-for="recipe in $store.state.recipes"
-          :key="recipe._id"
-          :recipe="recipe"
-        />
+  <main>
+    <AddRecipeForm />
+    <hr>
+    <h2 class="display-4">Recipes</h2>
+    <FilterRecipeForm />
+    <br>
+    <div class="font-italic">
+      <h2 v-if="$store.state.keyword && $store.state.ingredients.length">
+        Displaying recipes with keyword {{ $store.state.keyword }} and ingredient(s) {{ ingredients.join(',') }}
```

Comment:
Should `ingredients.join(',') ` be `$store.state.ingredients.join(', ')`?

Reply:
fixed